### DISCUSSION TITLE
大佬你好,请问现在是不是没有发送语音的功能

### DISCUSSION THREAD

**Discussion by Kamakoto** [Hoshikawa-Kaguya/Sora#31] [Q&A] [answered]
rt

**Comment by Kamakoto** [comment, marked as answer]
不好意思 已经找到了方法

**Reply by NingLiu1998** [reply]
分享一下呗

**Comment by Kamakoto** [comment]
原因是没有办法转换格式 看了一下CQHTTP的文档后发现需要安装ffmpeg

https://docs.go-cqhttp.org/guide/quick_start.html#%E5%AE%89%E8%A3%85-ffmpeg

**Comment by Yukari316** [comment]
![E8B6872CBC186109E71A39DBB380321A](https://user-images.githubusercontent.com/7535224/118687930-21d1a980-b838-11eb-82d1-a7d0a33b1267.gif)
文档确实我们还没写详细的进阶文档，如果有什么建议的话欢迎提出
